tests/Features/Skip.php

```diff
@@ -31,3 +35,19 @@
 it('skips when skip after assertion')
     ->assertTrue(true)
     ->skip();
+
+it('skips if the PHP version is the current one (with message)')
+    ->skipForVersion(PHP_VERSION, 'because of the current version')
```

Comment:
This is the broken test. 👍🏻 The 2nd character is the operator.
```suggestion
    ->skipForVersion(PHP_VERSION, null, 'because of the current version')
```